plugins/xpressive/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+INCLUDE(BuildPlugin)
+
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Dexprtk_disable_sc_andor -Dexprtk_disable_return -Dexprtk_disable_break_continue -Dexprtk_disable_comments -Dexprtk_disable_string_capabilities -Dexprtk_disable_rtl_io_file -Dexprtk_disable_rtl_vecops ${WERROR_FLAGS}")
+IF(WIN32 OR WIN64)
```

Comment:
[`WIN32`](https://cmake.org/cmake/help/v3.0/variable/WIN32.html) actually covers both cases, `WIN64` can safely be removed.